var/spack/repos/builtin/packages/py-numexpr3/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
```

Comment:
You don't mention this file in the PR title. Did this get pulled in by accident?

Reply:
Yes, I added another PR which got approved and that got included as well. Should I delete this and add a new PR ?